pkgs/development/python-modules/pysigma-backend-loki/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  pysigma,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "pysigma-backend-loki";
+  version = "0.12.3";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```

https://nixos.org/manual/nixpkgs/stable/#buildpythonpackage-parameters